website/docs/r/security_tag.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: security_tag"
+sidebar_current: "docs-vcd-resource-security-tag"
+description: |-
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+---
+
+# vcd\_security\_tag
+
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+
+Supported in provider *v3.7+* and requires VCD 10.3.1+
```

Comment:
?

```suggestion
Supported in provider *v3.7+* and requires VCD 10.3.0+
```

Reply:
Changed. I mixed things up with another feature that was only available for 10.3.1+